tinc.yaml

```diff
@@ -0,0 +1,3 @@
+dependencies:
+  - name: riak-protobuf
+    path: protobuf
```

Comment:
Are you using straight cabal to dev rather than stack? 

That's my preference too, though I do use this tool, [mafia](https://github.com/ambiata/mafia) as a wrapper for cabal that does what tinc does plus a few more things. It's become indispensable and gets copied into all my projects now.

My preference would be to just have the *standard* build tools in here.